UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs

```diff
@@ -1308,6 +1324,29 @@ public override void LocalTileReached(Vector3Int localPos)
 		//(Don't need to do this server side as the interactions are validated)
 		ControlTabs.CheckTabClose();
 	}
+
+	private void SyncMovementType(MovementType oldType, MovementType newType)
+	{
+		CurrentMovementType = newType;
+	}
+
+	private void UpdatePassables()
```

Comment:
You made sure to network this to client?

Reply:
yup, it is